runtime/src/lib.rs

```diff
@@ -1028,7 +1028,7 @@ parameter_type_with_key! {
 parameter_type_with_key! {
     pub GetChameleonPools: |base: AssetId| -> Option<(AssetId, sp_std::collections::btree_set::BTreeSet<AssetId>)> {
         if *base == common::XOR {
-            Some((common::KXOR, [common::ETH, common::VAL, common::PSWAP].into_iter().collect()))
```

Comment:
Do you need to completely revert this change or just deliver it later?
If later, please split it by feature flag.
If completely revert, please add coment to #1162 & #1163

Reply:
Fixed